compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -184,14 +184,14 @@ impl RegionHighlightMode {
 /// Trait for printers that pretty-print using `fmt::Write` to the printer.
 pub trait PrettyPrinter<'tcx>:
     Printer<
-        'tcx,
-        Error = fmt::Error,
-        Path = Self,
-        Region = Self,
-        Type = Self,
-        DynExistential = Self,
-        Const = Self,
-    > + fmt::Write
+    'tcx,
+    Error = fmt::Error,
+    Path = Self,
+    Region = Self,
+    Type = Self,
+    DynExistential = Self,
+    Const = Self,
+> + fmt::Write
```

Comment:
@calebcartwright 
Is this change intended? Looks like a regression.

AFAIK, the situation like
```
    HEADER BRACKET_OPEN
BRACKET_CLOSE
```
(with BRACKET_CLOSE being less indented than HEADER) is not supposed to happen in general.

Reply:
Thanks for the ping! This is definitely a bug that was introduced in v1.4.28, I'm guessing https://github.com/rust-lang/rustfmt/pull/4474 but will dig into it

Reply:
this has been fixed in source, should available within the next couple days